src/DB_Command.php

```diff
@@ -1102,6 +1103,79 @@ public function search( $args, $assoc_args ) {
+	 *     | post_date             | datetime            | NO   |     | 0000-00-00 00:00:00 |                |
+	 *     | post_date_gmt         | datetime            | NO   |     | 0000-00-00 00:00:00 |                |
+	 *     | post_content          | longtext            | NO   |     |                     |                |
+	 *     | post_title            | text                | NO   |     |                     |                |
+	 *     | post_excerpt          | text                | NO   |     |                     |                |
+	 *     | post_status           | varchar(20)         | NO   |     | publish             |                |
+	 *     | comment_status        | varchar(20)         | NO   |     | open                |                |
+	 *     | ping_status           | varchar(20)         | NO   |     | open                |                |
+	 *     | post_password         | varchar(255)        | NO   |     |                     |                |
+	 *     | post_name             | varchar(200)        | NO   | MUL |                     |                |
+	 *     | to_ping               | text                | NO   |     |                     |                |
+	 *     | pinged                | text                | NO   |     |                     |                |
+	 *     | post_modified         | datetime            | NO   |     | 0000-00-00 00:00:00 |                |
+	 *     | post_modified_gmt     | datetime            | NO   |     | 0000-00-00 00:00:00 |                |
+	 *     | post_content_filtered | longtext            | NO   |     |                     |                |
+	 *     | post_parent           | bigint(20) unsigned | NO   | MUL | 0                   |                |
+	 *     | guid                  | varchar(255)        | NO   |     |                     |                |
+	 *     | menu_order            | int(11)             | NO   |     | 0                   |                |
+	 *     | post_type             | varchar(20)         | NO   | MUL | post                |                |
+	 *     | post_mime_type        | varchar(100)        | NO   |     |                     |                |
+	 *     | comment_count         | bigint(20)          | NO   |     | 0                   |                |
+	 *     +-----------------------+---------------------+------+-----+---------------------+----------------+
+	 *
+	 * @when after_wp_load
+	 * @alias fields
+	 */
+	public function columns( $args, $assoc_args ) {
+		global $wpdb;
+
+		$format = WP_CLI\Utils\get_flag_value( $assoc_args, 'format' );
+
+		WP_CLI\Utils\wp_get_table_names( [ $args[0] ], [] );
+
+		// Get the database size.
+		$columns = $wpdb->get_results(
+			'SHOW COLUMNS FROM ' . $args[0]
+		);
+
+		// Display the rows.
+		$formatter_args = array(
+			'format' => $format,
+		);
+
+		$formatter = new Formatter($formatter_args, array( 'Field', 'Type', 'Null', 'Key', 'Default', 'Extra' ) );
```

Comment:
Given `$formatter_args` is built in a separate variable, the header fields should get the same treatment, especially as they produce a very long line here.